src/containers/scenes/mobile/nrc-landing-scene-mobile/nrc-landing-scene-mobile-constants.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { T } from '@transifex/react';
+
+export const NRC_LANDING_CARDS = [
```

Comment:
This wont work when we change language. We need a function that has to be run again when the locale changes